src/styles/block/stopwatch.scss

```diff
@@ -0,0 +1,79 @@
+.stopwatch {
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+
+  &__circle {
+    position: relative;
+    width: 500px;
+    height: 500px;
+    border-radius: 50%;
+    border: 3px dashed black;
+    background-color: aliceblue;
+
+    &--second {
```

Comment:
All these classes don't look like modifiers of the circle, they look like different elements